### ISSUE TITLE
Latest version not on pypi

### ISSUE THREAD

**Issue by guyrosin** [kuelumbus/rdkit-pypi#92]
Hey @kuelumbus,
I just noticed the latest version on [pypi](https://pypi.org/project/rdkit/#history) is still 2023.9.4
Thanks for your work!

**Comment by guyrosin** [comment]
Oh great, thank you (nothing to apologize for!)

**Comment by AlexHex7** [comment]
@kuelumbus  Hi. When will the latest version be uploaded in [pypi](https://pypi.org/project/rdkit/#history) ?

**Comment by kuelumbus** [comment]
The new RDKit version comes with stubs files (for functional autocompletion) that are not generate in the RDKit build process but in a post compilation step. Enabling the stubs build is not straightforward.  Please see  https://github.com/kuelumbus/rdkit-pypi/pull/93

Do you need the new version immediately?

**Comment by AlexHex7** [comment]
> The new RDKit version comes with stubs files (for functional autocompletion) that are not generate in the RDKit build process but in a post compilation step. Enabling the stubs build is not straightforward. Please see #93
> 
> Do you need the new version immediately?

No, I do not need it immediately.  Just to briefly mention it~ 
Thanks for your reply.
